policy-controller/grpc/src/outbound/grpc.rs

```diff
@@ -0,0 +1,240 @@
+use std::net::SocketAddr;
+
+use super::{convert_duration, default_balancer_config, default_queue_config};
+use crate::routes::{
+    convert_host_match, convert_request_header_modifier_filter, grpc::convert_match,
+};
+use linkerd2_proxy_api::{destination, grpc_route, http_route, meta, outbound};
+use linkerd_policy_controller_core::{
+    outbound::{Backend, Filter, OutboundRoute, OutboundRouteRule},
+    routes::{FailureInjectorFilter, GroupKindNamespaceName, GrpcRouteMatch},
+};
+
+pub(crate) fn protocol(
+    default_backend: outbound::Backend,
+    routes: impl Iterator<Item = (GroupKindNamespaceName, OutboundRoute<GrpcRouteMatch>)>,
+    accrual: Option<outbound::FailureAccrual>,
+) -> outbound::proxy_protocol::Kind {
+    let routes = routes
+        .map(|(gknn, route)| convert_outbound_route(gknn, route, default_backend.clone()))
+        .collect::<Vec<_>>();
+    outbound::proxy_protocol::Kind::Grpc(outbound::proxy_protocol::Grpc {
+        routes,
+        failure_accrual: accrual,
+    })
+}
+
+fn convert_outbound_route(
+    gknn: GroupKindNamespaceName,
+    OutboundRoute {
+        hostnames,
+        rules,
+        creation_timestamp: _,
```

Comment:
I like explicitly indicating which values we're discarding.  This helps remind us that this may need to be updated if we add more fields to OutboundRoute in the future.